src/main/java/serverutils/command/CmdSeekBlock.java

```diff
@@ -0,0 +1,118 @@
+package serverutils.command;
+
+import net.minecraft.block.Block;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.ChatComponentText;
+import net.minecraft.util.EnumChatFormatting;
+import net.minecraft.world.World;
+import net.minecraft.world.chunk.Chunk;
+import net.minecraft.world.gen.ChunkProviderServer;
+import net.minecraftforge.common.DimensionManager;
+
+import cpw.mods.fml.common.FMLLog;
+import serverutils.lib.command.CmdBase;
+
+public class CmdSeekBlock extends CmdBase {
+
+    private static final int MAX_RESULTS = 100;
```

Comment:
It would be nice to be able to set this in the command as well. Maybe like `/seek_block <blockID>:<meta> [max_results]` where it is bounded to/defaults to 100, but can be manually specified as lower if optionally provided